DEVELOPING.md

```diff
@@ -60,7 +60,7 @@ To run the API processes, you can use cargo:
 ```shell
 RUST_LOG=info cargo run -p trust -- vexination api --devmode -p 8081 &
 RUST_LOG=info cargo run -p trust -- bombastic api --devmode -p 8082 &
-RUST_LOG=info cargo run -p trust -- spog api -p 8083 &
+RUST_LOG=info cargo run -p trust -- spog api -p 8083 --authentication-issuer-url http://localhost:8090/realms/chicken --authentication-client-id frontend &
```

Comment:
Can we make these the defaults?

Reply:
Yes, I thought about making that a default when `--devmode` is set.